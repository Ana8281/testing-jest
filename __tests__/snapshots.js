//snapshots are data which are stored in file apart

const client = {
    name: 'Ana',
    balance: 500,
    type: "Premium"
}


describe('testing client', () => {
    test('Client is Premium', () => {
        expect(client).toMatchSnapshot()
    })

})


//for update the pbject the snapshot never muss be update in his file created better remove it 
//manually or through this command: npm t -- -u