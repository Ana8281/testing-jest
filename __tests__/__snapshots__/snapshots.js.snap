// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing client Client is Premium 1`] = `
Object {
  "balance": 500,
  "name": "Ana",
  "type": "Premium",
}
`;
